cpp/cmake/modules/FindBrotli.cmake

```diff
@@ -31,74 +31,76 @@
 #  BROTLI_SHARED_LIB, path to libbrotli's shared library
 #  BROTLI_FOUND, whether brotli has been found
 
-if( NOT "${BROTLI_HOME}" STREQUAL "")
-    file( TO_CMAKE_PATH "${BROTLI_HOME}" _native_path )
-    list( APPEND _brotli_roots ${_native_path} )
-elseif ( Brotli_HOME )
-    list( APPEND _brotli_roots ${Brotli_HOME} )
-endif()
+if( NOT WIN32 )
```

Comment:
I don't know why we need to `NOT WIN32`, if we add the libs to the `find_library` commands below it worked fine for me, though based on the builds it sounds like the default finders are fine.